tests/test_basic_s3.py

```diff
@@ -37,8 +43,14 @@ async def test_can_make_request_no_verify(s3_client):
     assert actual_keys == ['Buckets', 'Owner', 'ResponseMetadata']
 
 
+@pytest.fixture
+def skip_httpx(current_http_backend: str) -> None:
+    if current_http_backend == 'httpx':
+        pytest.skip('proxy support not implemented for httpx')
```

Comment:
better put this in conftest.py

Reply:
why did I ever make it a fixture in the first place?? :sweat_smile:

Reply:
oh, because it fails in the setup of `test_fail_proxy_request` so I can't skip in the test body. It's specifically only used by `test_fail_proxy_request` so I think it should be in this file, but I'll add a comment

Reply:
we could insert a bunch of logic into the `s3_client` fixture or somewhere, but I think the logic for skipping should be next to the test it skips. If it was used for skipping several other tests as well I'd agree though

Reply:
this may be useful later for other tests, so still think it would be nice in conftest

Reply:
not a hard req tho, I'm just worried later someone writing another one of these in another file due to lack of discoverability

Reply:
ah, I see single-use fixtures like `aa_fail_proxy_config` are currently placed in conftest, so if they're there then this one can def go there